src/main/java/com/google/api/gax/grpc/PageStreamingCallSettings.java

```diff
@@ -58,10 +58,8 @@
     return createBaseCallable(channel, executor);
   }
 
-  /**
-   * Package-private, for use by ApiCallable.
-   */
-  ApiCallable<RequestT, PageAccessor<ResourceT>> createPagedVariant(
+  /** Package-private, for use by ApiCallable. */
+  ApiCallable<RequestT, PagedListResponse<RequestT, ResponseT, ResourceT>> createPagedVariant(
```

Comment:
I lament the increased number of parameterized types when going from PageAccessor -> PagedListResponse...